### PR TITLE
refactor(rxDevicePaths): rename devicePaths with rx prefix

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/FRMW-1165

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

**Comment by cathysiller** [comment]
Dev LGTM

File: src/utilities/rxDevicePaths/scripts/rxDevicePaths.spec.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        });
+    });
+
+    it('should exist', function () {
+        expect(dp).to.exist;
+    });
+});
+
+// TODO: Deprecated. Remove in 4.0.0
+describe('devicePaths', function () {
```
Comment (CITguy) [inline src/utilities/rxDevicePaths/scripts/rxDevicePaths.spec.js]: Add `(DEPRECATED)` to describe text.

**Comment by parlarjb** [comment]
LGTM

**Comment by halkon** [comment]
LGTM

**Pull request merged**